src/vm/finalizerthread.cpp

```diff
@@ -638,6 +670,20 @@ VOID FinalizerThread::FinalizerThreadWorker(void *args)
 
         WaitForFinalizerEvent (hEventFinalizer);
 
+#ifdef FEATURE_PAL
+        if(g_TriggerHeapDump && (CLRGetTickCount64() > (LastHeapDumpTime + 10000)))
```

Comment:
can you make this a #define, eg #define PAL_HEAP_DUMP_TIME_OUT 100000 since you are using it in quite a few places?